public/index.html

```diff
@@ -40,6 +40,6 @@
 </div>
 
 <div id="message-bar" class="swagger-ui-wrap">&nbsp;</div>
-<div id="swagger-ui-container" class="swagger-ui-wrap"></div>
+<div id="swagger-ui-container" class="swagger-ui-wrap" style="padding: 20px"></div>
```

Comment:
Will be fixed by https://github.com/strongloop/strong-swagger-ui/pull/6, I have removed this change from this patch.